File: ueueo-framework/ueueo-blob-storing/src/main/java/com/ueueo/BlobStoring/BlobNormalizeNaming.cs
namespace Volo.Abp.BlobStoring;

public class BlobNormalizeNaming
{
    public String ContainerName;//  { get; }

    public String BlobName;//  { get; }

    public BlobNormalizeNaming(String containerName, String blobName)
    {
        ContainerName = containerName;
        BlobName = blobName;
    }
}

File: ueueo-framework/ueueo-blob-storing/src/main/java/com/ueueo/blobstoring/BlobNormalizeNaming.java
package com.ueueo.blobstoring;

public class BlobNormalizeNaming {
    public String containerName;

    public String blobName;

    public BlobNormalizeNaming(String containerName, String blobName) {
        this.containerName = containerName;
        this.blobName = blobName;
    }

    public String getContainerName() {
        return containerName;
    }

    public String getBlobName() {
        return blobName;
    }
}
